automol/embed/_dgeom.py

```diff
@@ -59,13 +65,16 @@ def sample_raw_distance_coordinates(lmat: MatrixLike, umat: MatrixLike, dim4=Tru
     return xmat
 
 
-def triangle_smooth_bounds_matrices(lmat: MatrixLike, umat: MatrixLike):
+def triangle_smooth_bounds_matrices(lmat: MatrixLike, umat: MatrixLike) -> MatrixLike:
```

Comment:
` -> numpy.ndarray`

Reply:
Throughout, `MatrixLike` return types should be replaced with `numpy.ndarray` to be more specific.

Reply:
In general, the philosophy is to be more general for input types (allowing flexibility for how the function is called), but as specific as possible for return types (telling the user exactly what they are getting).